### ISSUE TITLE
"Error deleting sandbox folder" (CANNOT DELETE ANY SANDBOX) 

### ISSUE THREAD

**Issue by unoukujou** [sandboxie-plus/Sandboxie#4435]
### Describe what you noticed and did

No idea why but I'm unable to delete any sandbox.

Always this message:

![20241218_imgpipe_2](https://github.com/user-attachments/assets/835a041f-2c0a-418a-93f7-3362bb30711b)

v1.15.3
Windows 11

Not doing anything out of the ordinary. No idea why it's happening.

After it fails to delete, if I go and browse the actual sandbox location, all the files are there, but for some reason the sandbox folder gets reverted to the default Windows folder icon instead of the sandboxie folder icon. I'm not sure if that means anything but just figured I'd mention that this does happen when I try to delete a sandbox. So I get the error that it can't delete the sandbox and the folder of the sandbox turns back to the Windows default icon for folders. The sandbox is not deleted.

I can, however, delete the sandbox folder manually without any problem. Not sure why this is happening, but if you have any ideas, please let me know.

### How often did you encounter it so far?

whenever I try to delete a sandbox.

### Expected behavior

sandbox should be deleted

### Affected program

Not relevant.

### Download link

Not relevant.

### Where is the program located?

Not relevant to my request.

### Did the program or any related process close unexpectedly?

No, not at all.

### What version of Sandboxie are you running now?

Sandboxie Plus 1.15.3 64bit windows 11

### Is it a new installation of Sandboxie?

I recently did a new clean installation.

### In which sandbox type you have this problem?

Not relevant to my request.

### Can you reproduce this problem on a new empty sandbox?

I can confirm it also on a new empty sandbox.

### What is your Windows edition and version?

Windows 11 64 bit

### In which Windows account you have this problem?

A local account (Standard user).

### Please mention any installed security software

Microsoft Defender Antivirus. Nothing else. Also, I have the whole Sandbox folder excluded

**Comment by unoukujou** [comment]
Also to add to this, I just want to say that all the sandboxed processes are terminated. I don't have any running processes in any of the sandboxes. So it's not a problem like I have an active process and therefore that's why it can't be deleted. It's not anything like that.

**Comment by unoukujou** [comment]
Also, it's a regular local internal hard drive, not a removable or a flash drive or a USB drive or share or anything like that.

**Comment by offhub** [comment]
This issue has been fixed in the next version (1.15.4), but it has not been released yet. If you don't want to wait for the new version to be released, you can follow the steps below to resolve the issue.

---

1. Download the file from the link below and extract it (GitHub login required):
   
   - For 64-bit systems:  
    ```
    https://github.com/sandboxie-plus/Sandboxie/actions/runs/12080928142/artifacts/2252720657
    ```

2. Close the Sandboxie-Plus interface.

3. Navigate to the folder where the files were extracted and copy the **QSbieAPI.dll** file to the Sandboxie-Plus installation directory.

4. Open the Sandboxie-Plus interface and verify if the deletion process is functioning correctly.

**Comment by unoukujou** [comment]
> This issue has been fixed in the next version (1.15.4), but it has not been released yet. If you don't want to wait for the new version to be released, you can follow the steps below to resolve the issue.
> 
> 1. Download the file from the link below and extract it (GitHub login required):
>    
>    * For 64-bit systems:
>    
>    ```
>    https://github.com/sandboxie-plus/Sandboxie/actions/runs/12080928142/artifacts/2252720657
>    ```
> 2. Close the Sandboxie-Plus interface.
> 3. Navigate to the folder where the files were extracted and copy the **QSbieAPI.dll** file to the Sandboxie-Plus installation directory.
> 4. Open the Sandboxie-Plus interface and verify if the deletion process is functioning correctly.

Thanks. I can confirm, the new QSbieAPI.dll solves the problem.

**Comment by offhub** [comment]
[1.15.4 is released](https://github.com/sandboxie-plus/Sandboxie/releases/latest)
